### PR TITLE
Add React Hooks

### PR DESCRIPTION
**Pull request by glauberfc**
**[Awesome React Hooks](https://github.com/glauberfc/awesome-react-hooks)**

**A list about React Hooks new feature with very good links about this subject**

PR reviews:
#1482
#1451

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -191,6 +191,7 @@
 	- [Protips](https://github.com/AllThingsSmitty/css-protips#readme)
 - [React](https://github.com/enaqx/awesome-react#readme) - App framework.
 	- [Relay](https://github.com/expede/awesome-relay#readme) - Framework for building data-driven React apps.
+	- [React Hooks](https://github.com/glauberfc/awesome-react-hooks#readme) - A new feature that lets you use state and other React features without writing a class.
```
Comment (dend) [inline readme.md]: Should probably come before Relay, if you are adding this in alphabetical order.

Comment (glauberfc) [reply readme.md]: Hmm...Are this alphabetical order required @sindresorhus

Comment (sindresorhus) [reply readme.md]: If you have to ask, you have clearly not read the PR guidelines:

> By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Comment by sindresorhus** [comment]
> Includes a succinct description of the project/theme at the top of the readme. (Example)

This should describe what hooks are.

---

> Includes a project logo/illustration whenever possible.
**The image should link to the project website or any relevant website.**
**The image should be high-DPI. Set it to maximum half the width of the original image.**

**Comment by glauberfc** [comment]
I think the errors were fixed @sindresorhus

**Comment by sindresorhus** [comment]
> The image should link to the project website or any relevant website.

Linking to the list makes no sense.

**Comment by glauberfc** [comment]
Now the cover links to: [React Hooks intro](https://reactjs.org/docs/hooks-intro.html).

**Comment by sindresorhus** [comment]
Tweet: https://twitter.com/awesome__re/status/1128226625550598145

**Pull request merged**